mentoring/settings.py

```diff
@@ -181,6 +182,11 @@
 
 USE_TZ = True
 
+# content security policy
+# NOTE: unsafe-eval is only required for development builds of the react app (`yarn dev`, not `yarn build`)
+CSP_DEFAULT_SRC = ["'self'", "'unsafe-inline'", "'unsafe-eval'"]
```

Comment:
One way to implement this comment in code is to trigger off the ``DEBUG`` setting:
```suggestion
CSP_DEFAULT_SRC = ["'self'", "'unsafe-inline'"]
# For development builds of the react app (`yarn dev`, not `yarn build`), eval is needed
if DEBUG:
    CSP_DEFAULT_SRC.append("'unsafe-eval'")
```

Later, when ``DEBUG`` is loaded from the environment (default ``False`` for prod) you'll automatically get a tighter CSP for production. However, there's a few production overrides (``SECRET_KEY``, etc.), we could do that work when we get to production configuration (and may have a better trigger than ``DEBUG``).